pom.xml

```diff
@@ -88,7 +88,8 @@
         <scylladb.version>1.7.1</scylladb.version>
         <!-- align with org.apache.hbase:hbase -->
         <jackson1.version>1.9.13</jackson1.version>
-        <jackson2.version>2.12.2</jackson2.version>
+        <!-- align with org.apache.spark:spark-core_2.12 -->
```

Comment:
We should upgrade spark in a next pr to 3 or not?

Reply:
Not sure if I got you. spark-gremlin 3.5.0 depends on org.apache.spark:spark-core_2.12 3.0.0: https://mvnrepository.com/artifact/org.apache.tinkerpop/spark-gremlin/3.5.0

Reply:
2.12 for jackson or not?

Reply:
The spark package name is org.apache.spark:spark-core_2.12. The version of it is 3.0.0. It requires Jackson 2.10.0, otherwise, it throws runtime exceptions.

Reply:
It looks like the 2.12 is for the support scale version.

Reply:
You mean scala right? Yes, the "2.12" suffix of "spark_core" is seemingly to match the scala version.